### DISCUSSION TITLE
Pensar como organizar layouts (ex: Figma)

### DISCUSSION THREAD

**Discussion by Flamilani** [techexpertspro/portal-ascepa#23] [Ideas]
Temos que pensar como organizar telas antes de desenvolver componentes

**Comment by FelipeDuarteLuna** [comment]
Bom dia Flamilani,

Concordo com seu raciocinio, um dev que não sabe o que codar é complicado. Depois quando sugir a ideia dos @andrewarosario  e @paoloalmeida , podemos chamar a @Rosanaxica ela é Dev e faz trampo de UX. No nosso projeto de conclusão da Mentoria 1.0 ela fez o UX e eu fiquei apoiando. 

https://github.com/FelipeDuarteLuna/painel-administrativo

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/6182f1de-d5f2-4a0b-b0c6-0f7e0fa824d5)

**Reply by Flamilani** [reply]
Que ótima iniciativa!

**Comment by andrewarosario** [comment]
Bom dia galera!
No momento temos a @ortegavan que vai nos ajudar na prototipação. Ela manja muito! 🚀

**Reply by ortegavan** [reply]
Obrigada pela menção, mestre! Duvidinha: vamos fazer alguns wireframes ou vamos direto para os protótipos de alta?

**Reply by andrewarosario** [reply]
Nessa escolha acho que você tem muito mais autonomia que a gente hahaha o que você acha que seria melhor?

**Reply by ortegavan** [reply]
Particularmente, gosto de ter alguns wireframes (não precisa ser tudo) para diagramarmos o layout primeiro sem muita preocupação com tudo o que o design de alta exige, acho que eles ajudam a clarear as ideias iniciais. E ajudam, inclusive, a desenvolver o protótipo de forma mais rápida depois. É um tempinho a mais no começo mas que só traz ganho depois... eu curto! =)

**Reply by andrewarosario** [reply]
Perfeito! Bora seguir essa estratégia 💯

**Comment by ortegavan** [comment]
Amigos, segue link dos wireframes no Figma: https://www.figma.com/design/3qawe9F2BwNxmGG87aWBzn/Portal-Ascepa?node-id=1-3&t=gKtp3e6fwukB76PX-1
